flocx_market/templates/layout.html

```diff
@@ -0,0 +1,22 @@
+<html>
```

Comment:
Same here: unnecessary for this project.

Reply:
removed and made a new PR